dao/src/main/scala/za/co/absa/enceladus/dao/rest/AuthClient.scala

```diff
@@ -61,7 +61,7 @@ sealed abstract class AuthClient(username: String, restTemplate: RestTemplate, a
       val response = requestAuthentication(url(baseUrl))
       val statusCode = response.getStatusCode
 
-      lazy val errMessage = s"Authentication failure ($statusCode): $username"
+      def errMessage = s"Authentication failure ($statusCode): $username"
```

Comment:
Customary the embedded/nested function are defined right under the signature of the parent function (yes that would require this one to have at least one input parameter of `statusCode`

Reply:
All right, thank you. In that case I feel like it's not worth the effort, the benefit is too small since it's really just a string, that doesn't involve any demanding computation.